### DISCUSSION TITLE
Currydouble Curtain Switch (5th Gen)

### DISCUSSION THREAD

**Discussion by LuisMarinheiro** [arendst/Tasmota#15491] [Q&A] [answered]
Hi Guys

I currently flashed some Currydouble Curtain Switches (UMI002) (4th Gen) following the instructions on
https://templates.blakadder.com/currydouble_UMI002.html 
and 
https://github.com/arendst/Tasmota/discussions/12634

After all instructions, all looks working well.

I also have the most recent versions of Currydouble Curtain Switches (5th Gen), and I was able to take a risk to flash these new ones with tasmota. (if it's possible)
Has anyone tried or has advice on how to flash these versions? Or if tasmota may be compatible with these switches?

I put here some pictures:
![20220430_115827](https://user-images.githubusercontent.com/7900457/166102942-4d2e414f-ca04-41f7-ae45-761297bfade4.jpeg)

![20220430_115837](https://user-images.githubusercontent.com/7900457/166102950-8fc47368-4f15-4155-ab64-34dd7338e849.jpeg)

From what I could understand it contains the module WB3S, and if I'm not wrong we could find detailed information about this module here:
https://developer.tuya.com/en/docs/iot/wb3s-module-datasheet?id=K9dx20n6hz5n4

**Comment by barbudor** [comment, marked as answer]
Brain transplant with a ESP12F is the way
https://tasmota.github.io/docs/Pinouts/#esp-12eesp-12f
